src/constants.py

```diff
@@ -39,9 +38,15 @@
 
 DEMO_TOKEN = '45372338e04f5a41af949024db929d46'
 
-PROBLEMATIC_PACKAGES = [
+BLACKLISTED_PACKAGES = [
     # 'Pyrender', 'Trimesh',
-    'moderngl', 'pyopengl', 'pyglet', 'pythreejs', 'panda3d', # because they need a screen,
+    'moderngl', 'pyopengl', 'pyglet', 'pythreejs', 'panda3d',  # because they need a screen,
+    'tika',  # because it needs java
+    'fastapi', 'uvicorn', 'starlette',  # because we use jina executors
 ]
 
-UNNECESSARY_PACKAGES = ['fastapi']
```

Comment:
I think we should differentiate between packages which make an entire combination of packages irrelevant (as the package mustn't be used but it is functionally needed to accomplish the task) and between packages which should be skipped as they are not needed (e.g. fastapi, flask, etc.)

Reply:
good point. I updated it.